solutions/src/main/java/ru/sberbank/school/task02/exception/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,18 @@
+package ru.sberbank.school.task02.exception;
+
+import ru.sberbank.school.task02.ExtendedFxConversionService;
+import ru.sberbank.school.task02.ExternalQuotesService;
+import ru.sberbank.school.task02.FxConversionService;
+import ru.sberbank.school.task02.ServiceFactory;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+  @Override
+  public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    return new FxConversionServiceImpl(externalQuotesService);
+  }
+
+  @Override
+  public ExtendedFxConversionService getExtendedFxConversionService(ExternalQuotesService externalQuotesService) {
+    return null;
```

Comment:
&#39;method def&#39; child has incorrect indentation level 4, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a69847846>*